src/test/java/com/checkmarx/jira/JiraTestUtils.java

```diff
@@ -340,7 +366,7 @@ private ObjectNode getProjectRequestBody(String key) throws JsonProcessingExcept
                 " 'projectTypeKey': 'software'," +
                 " 'projectTemplateKey': 'com.pyxis.greenhopper.jira:gh-scrum-template'," +
                 " 'description': 'Automation'," +
-                " 'lead': 'admin'," +
+                " 'leadAccountId': '5e3815a9e697e80e5b414719'," +
```

Comment:
What is this magic number? Can we put it into a constant?

Reply:
This replaces the "admin" that we had before. this is the account id of our test user, as the field name implies.
Well, I think that it can stay like that (like it was before), but if it really bothers I can change it of course.

Reply:
Let's put it into TEST_USER_ACCOUNT_ID constant.